packages/accounts-password/password_tests.js

```diff
@@ -410,6 +410,13 @@ if (Meteor.isClient) (function () {
       "email", [
     createUserStep,
     logoutStep,
+    // Create user error without callback should not throw error
+    function (test, expect) {
```

Comment:
Why did you add this new step? I think other tests cover this situation, no? Plus it fails because the password is empty.